src/core/Akka.Remote/Endpoint.cs

```diff
@@ -320,44 +339,47 @@ internal class ReliableDeliverySupervisor : UntypedActor
         /// If we already have an inbound handle then UID is initially confirmed.
         /// (This actor is never restarted.)
         /// </summary>
-        public bool UidConfirmed { get; set; }
+        public bool UidConfirmed { get; private set; }
 
-        public Deadline BailoutAt
+        private Deadline _bailoutAt = null;
+
+        protected override SupervisorStrategy SupervisorStrategy()
         {
-            get { return Deadline.Now + _settings.InitialSysMsgDeliveryTimeout; }
+            return new OneForOneStrategy(ex =>
+            {
+                if (ex is IAssociationProblem)
```

Comment:
Nope, not a bug-  reverted it. I misread the JVM source when I left that comment originally.